### ISSUE TITLE
Printing of warning message on unknown package should be configurable

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-industrial/universal_robot#28]
The warning message introduced in 04197800 is impossible to disable in the current implementation. Ideally, the user would be able to suppress it using a configuration parameter (either via a launchfile or through support for dynamic reconfigure).

Perhaps warning interval should also be configurable (currently hard coded at 1 second).

**Comment by abubeck** [comment]
@gavanderhoorn: Is this still relevant, as deserialize has been rewritten?

**Comment by gavanderhoorn** [comment]
Yes, it is still an issue. Any new pkt ids that are added to the protocol used on the secondary interface will result in warnings printed on the console.

`deserialize.py` still stores unknown pkts ([here](https://github.com/ros-industrial/universal_robot/blob/3171fe11105264aa1172936e77f05e16efcbbfff/ur_driver/src/ur_driver/deserialize.py#L355)), and `driver.py` still prints warnings ([here](https://github.com/ros-industrial/universal_robot/blob/6c60789e0fe443f8483af649cf2f2314e43c7b3c/ur_driver/src/ur_driver/driver.py#L250-L255)).

If we are ok with the 1 second interval, then this can be closed. But unknown pkt IDs will result in a msg printed every second in that case (as the unknown pkt types are likely to be part of every `ROBOT_STATE` msg being received).

I'll see if I can implement a Python version of `ROS_DEBUG_ONCE(..)`. Perhaps that would be better.

**Comment by abubeck** [comment]
To be honest I prefer the annoying version with the 1 sec interval :) Unknown pkts tell us there is something wrong/not in sync and we should fix it.

**Comment by gavanderhoorn** [comment]
Well the idea behind PR #23 was to make the driver just ignore unknown pkts, instead of crashing. That makes it possible for users to use the driver even if it isn't completely up-to-date with a new (Polyscope) software version.

In order to let the user know what was going on, I introduced the warning. But later I realised that warning will be printed every second, which seems unnecessary but also floods the terminal, which drowns out all other console messages. From a usability stand point, that is not very nice.

Only new pkts are ignored, changes to old ones will still (likely) crash the driver, so a single (or a few) warnings saying something like "Ignoring unknown pkt types X, Y and Z. This warning will only be printed once." seems more appropriate.
